code-highlighter.html

```diff
@@ -70,19 +70,18 @@
         pointer-events: none;
         z-index: 0;
         right: 0;
-        opacity: 0.2;
       }
 
       #diffColors div {
         height: var(--line-height, 24px);
       }
 
-      #diffColors .add {
-        background-color: var(--add-color, green);
+      #diffColors .add, .icon-add{
+        background-color: var(--add-color, rgba(0,128, 0, 0.2));
       }
 
-      #diffColors .del {
-        background-color: var(--del-color, red);
+      #diffColors .del, .icon-del {
+        background-color: var(--del-color, rgba(255,0, 0, 0.2));
```

Comment:
This was needed, because the opacity works trough for it's children (even if a child has a set opacity of 1 it's limited to the parent value).
Setting the opacity for the background color itself does not.